application.yml

```diff
@@ -54,6 +54,19 @@ management:
       prometheus:
         enabled: true
 
+# Chaos Engineering
+management.endpoint.chaosmonkey.enabled: true
+chaos:
+  monkey:
+    enabled: true
+    watcher:
+      component: false
+      controller: false
+      # Defaults to potential issues with repositories - can be changed via REST API
```

Comment:
Does this issue is referenced in the Chaos Monkey GitHub project?

Reply:
Ah it's a copy paste from my pet project, I'll remove it